### DISCUSSION TITLE
The Vaultwarden container, set up as a reverse proxy, is unable to establish a proper connection to the backend.

### DISCUSSION THREAD

**Discussion by rayangl** [dani-garcia/vaultwarden#5079] [Q&A]
When utilizing [lucky](https://github.com/gdy666/lucky) for reverse proxying, there is an issue where the backend cannot be accessed, whereas other reverse proxy setups are working correctly.
![image](https://github.com/user-attachments/assets/0b3b7ff5-6ea4-4816-829b-3164aa18543a)
![image](https://github.com/user-attachments/assets/0d212969-7ee1-438a-8989-e6437a9b2ce5)
![image](https://github.com/user-attachments/assets/a3f13571-82e8-4f6b-bcc6-a9405a7d6b81)

**Comment by BlackDex** [comment]
Looks like a `lucky` issue. And something in regards to ipv6 and ipv4 mixing or something.

As i have no clue what `lucky` is and how it works. I can't help you there, but maybe someone else does.
